File: client/src/orion_spur/common/material/RemoteObjectListDto.java
package orion_spur.common.material;

import java.util.List;

public class RemoteObjectListDto 
{
	private List<RemoteObjectDto> RemoteObjectList;

	public RemoteObjectListDto(){}

	public RemoteObjectListDto(List<RemoteObjectDto> RemoteObjectList)
	{
		this.RemoteObjectList = RemoteObjectList;
	}

	public List<RemoteObjectDto> getRemoteObjectList()
	{
		return RemoteObjectList;
	}
}

File: server/src/material/RemoteObjectListDto.go
package material

type RemoteObjectListDto struct {
	RemoteObjectList  []RemoteObjectDto `json:"RemoteObjectList"`
}

func NewRemoteObjectListDto(remoteObjectList []RemoteObjectDto)RemoteObjectListDto{
	return RemoteObjectListDto{RemoteObjectList: remoteObjectList}
}

func (r RemoteObjectListDto) GetRemoteObjectList() []RemoteObjectDto {
	return r.RemoteObjectList
}


